rmw_fastrtps_cpp/QUALITY_DECLARATION.md

```diff
@@ -85,7 +85,7 @@ The results of the test can be found [here](https://ci.ros2.org/view/nightly/job
 
 ### Feature Testing [4.i]
 
-There is no feature testing in `rmw_fastrtps_cpp`.
+All `rmw_fastrtps_cpp` public features are ROS middleware features. Integration and system tests up the stack provide coverage.
```

Comment:
Be more specific about what "up the stack" means. Also in this case, since this package provides a lot of functionality that's depended on by other packages, I would provide explicit links to the packages and CI links where this feature testing is happening. Essentially, we're testing rmw_fastrtps_cpp all the time by making it a default configuration on ci.ros2.org and build.ros2.org. So it would be good to just state that that's how we know this is very carefully vetted.

Reply:
See https://github.com/ros2/rmw_fastrtps/pull/380/commits/4e7316ad6346f9e7a7253ae1ee81961ac346092c